include/zephyr/net/wifi_mgmt.h

```diff
@@ -139,6 +140,9 @@ enum net_event_wifi_cmd {
 
 #define NET_EVENT_WIFI_RAW_SCAN_RESULT                          \
 	(_NET_WIFI_EVENT | NET_EVENT_WIFI_CMD_RAW_SCAN_RESULT)
+
+#define NET_EVENT_WIFI_DISCONNECT_COMPLETE			\
```

Comment:
Where is this one used? Is it that with this change we will still receive the NET_EVENT_WIFI_DISCONNECT_RESULT in the application with the difference being now we block the disconnect command until it really succeeds?

Reply:
No, this PR only adds the event, this is sent https://github.com/nrfconnect/sdk-hostap/pull/94/files#diff-ff43beace1c9eb2129211e73d03f9385752af39391aa954185b48b9891e1c9b4R305

Reply:
Turned out the `NET_EVENT_WIFI_CMD_DISCONNECT_COMPLETE` was really not used :) Thanks for the change